exercises/practice/zebra-puzzle/.meta/example.h

```diff
@@ -0,0 +1,19 @@
+#if !defined(ZEBRA_PUZZLE_H)
+#define ZEBRA_PUZZLE_H
+
+#include <string>
+
+namespace zebra_puzzle 
+{
+
+typedef struct 
+{
+    std::string drinksWater;
+    std::string ownsZebra;
+} Solution;
```

Comment:
`typedef struct { ... } type_name;` is a C idiom.  
In C++ you can write `struct type_name { ... };`, just like you did with `House` in the `.cpp` file.